packagers/pnpm-workspaces-custom-output/workspace/sub-project-1/prisma/schema.prisma

```diff
@@ -0,0 +1,15 @@
+generator client {
+  provider = "prisma-client-js"
+  output = "../node_modules/.prisma/client"
```

Comment:
Wouldn't it be better to use a non standard folder for the custom output?

Reply:
I think that we could add another test. With this one I want to make sure that we would not generate in the root `node_modules`, basically testing that it properly gets generated relatively to where we called the `cli`.